third_party/move/move-compiler-v2/tests/bytecode-generator/v1-typing/global_invalid.exp

```diff
@@ -17,23 +17,23 @@ module 0x42::m {
 
 
 Diagnostics:
-error: Expected a struct type. Global storage operations are restricted to struct types declared in the current module. Found: '#0'
+error: Expected a struct type. Global storage operations are restricted to struct types declared in the current module. Found: 'T'
   ┌─ tests/bytecode-generator/v1-typing/global_invalid.move:4:17
   │
 4 │         assert!(exists<T>(addr), 0);
   │                 ^^^^^^^^^^^^^^^
   │                 │
   │                 Invalid call to exists<#0>.
```

Comment:
There is still `#0` used on line 26?

Reply:
Thanks for catching this. Updated the fix in this commit: https://github.com/aptos-labs/aptos-core/pull/12085/commits/7345b790d9d1a686c4dffcd99837e346832a2086